mujoco_warp/_src/collision_driver_test.py

```diff
@@ -16,6 +16,7 @@
 
 import mujoco
 import numpy as np
+import pytest
```

Comment:
we can remove this import and instead use `self.skipTest` below. example: https://github.com/google-deepmind/mujoco_warp/blob/3f1197edd97a96a21384fcb418aa566306a22654/mujoco_warp/_src/jax_test.py#L38